### PR TITLE
user_guide: improve section workflow-configuration

### PR DESCRIPTION
**Pull request by bertsky**
- mention `ocrd-import` option for _non-existing METS_ case
- recommend `-h` instead of just `-J` for _more information about processors_
- fix usage of `workflow-configuration`

### CODE REVIEW

File: site/en/user_guide.md [hunk 1]
```diff
@@ -169,6 +169,12 @@ for i in OCR-D-IMG/*.tif; do base=`basename ${i} .tif`;docker run --rm -u $(id -
 
 In the end, your METS file should look like this [example METS](example_mets.md).
 
+Alternatively, `ocrd-import` from [workflow-configuration](#workflow-configuration) is a shell script which does all of the above (and can convert arbitrary image formats) automatically. For usage options, see:
+
+```sh
+ocrd-import -h
```
Comment (EEngl52) [inline site/en/user_guide.md:175]: could you please add a basic sample call for ocrd-import to make it useable right away?

File: site/en/user_guide.md [hunk 2]
```diff
@@ -414,9 +433,9 @@ Type 'ocrd-' followed by `TAB` to get a list of all available processors.
 To get further information about one processor type
 
 ```sh
-[name_of_selected_processor] -J
+[name_of_selected_processor] -h
```
Comment (EEngl52) [inline site/en/user_guide.md:436]: I wouldn't leave out JSON completely. Perhaps we could mention it after help?

**Comment by EEngl52** [comment]
thanks a lot for your emendations @bertsky ! this section was really lacking some important details

File: site/en/user_guide.md (hunk 1 above)
Comment (bertsky) [reply site/en/user_guide.md:176]: Done!

File: site/en/user_guide.md (hunk 2 above)
Comment (bertsky) [reply site/en/user_guide.md:445]: `-h` includes everything from `-J` as well (but puts it in context)

**Pull request merged**